scripts/repository.lic

```diff
@@ -1717,7 +1774,7 @@ elsif (cmd[0] =~ /^delete$/i)
     echo "error: file not found"
   elsif list.length > 1
     list.unshift(headers)
-    respond "\nMultiple files match: be more specific\n\n#{format_list.call(list)}\n\n"
+    puts "\nMultiple files match: be more specific\n\n#{format_list.call(list, 'delete')}\n\n"
```

Comment:
do NOT use `puts` as it has known issues with terminal based FEs. In order to send actual XML, change the `respond` to `_respond`